lib/commands/exam.js

```diff
@@ -101,6 +116,47 @@ module.exports = TestCommand.extend({
     return this._super.run.apply(this, arguments);
   },
 
+  /**
+   *
+   * Instantiate an executionMappingObj object
+   *
+   * @param {Object} option
+   */
+  getExecutionMappigInstance: function(options) {
+    if (executionMappingObj !== undefined && executionMappingObj['numberOfBrowsers'] !== undefined && executionMappingObj['executionMapping'] !== undefined) {
```

Comment:
Could we not use Hungarian notation here for `executionMappingObj`? I think just dropping the `Obj` would suffice.